src/main/java/core/basesyntax/animals/Aviaries.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.animals;
+
+import core.basesyntax.kind.Birds;
+import core.basesyntax.kind.Fishes;
+import core.basesyntax.kind.LandAnimal;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Aviaries {
```

Comment:
make all class names singular
```suggestion
public class Aviary {
```